pipenv/project.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
 import codecs
+import io
```

Comment:
`io.open` was needed for consistent 2/3 behavior without using the deprecated `'U'` mode.  Given the discussion in #1847 it sounds like this may affect the encoding situation as well.